azure_jumpstart_ag/artifacts/PowerShell/Modules/manufacturing.psm1

```diff
@@ -531,4 +530,25 @@ function Configure-MQTTIpAddress {
             netsh interface portproxy add v4tov4 listenport=1883 listenaddress=0.0.0.0 connectport=1883 connectaddress=$using:mqttIp
         }
     }
+    return $mqttIp
```

Comment:
This will only return the Ip of one of the clusters, we need to have an array and add the Ip of every cluster then return the array

Reply:
Done, thanks for this call out